### PR TITLE
Add Freedompro doc

### PR DESCRIPTION
**Pull request by stefano055415**
## Proposed change

This is a new feature request to integrate a new platform called Freedompro

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/46332
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2253
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @stefano055415, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16492).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/freedompro.markdown [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ha_category:
+  - Sensor
+  - Binary Sensor
+  - Climate
+  - Light
+  - Fan
+  - Lock
+  - Cover
+  - Switch
+featured: true
```
Comment (frenck) [inline source/_integrations/freedompro.markdown]: This is not up to you to decide 😉 
```suggestion
```

File: source/_integrations/freedompro.markdown [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+description: Instructions for how to integrate Freedompro accessories within Home Assistant.
+ha_category:
+  - Sensor
+  - Binary Sensor
+  - Climate
+  - Light
+  - Fan
+  - Lock
+  - Cover
+  - Switch
```
Comment (frenck) [inline source/_integrations/freedompro.markdown]: The initial PR should be limited to a single platform.

File: source/_integrations/freedompro.markdown [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+| Garage Door          |        Cover            |
+| Door                 |        Cover            |
+| Window               |        Cover            |
+| Fan                  |        Fan              |
+| Light                |        Lightbulb        |
+| Lock                 |        Lock             |
+| Switch               |        Switch           |
+| Switch               |        Outlet           |
+
+<div class='note'>
```
Comment (frenck) [inline source/_integrations/freedompro.markdown]: Why is this a big note box and not part of the preliminary steps?

File: source/_integrations/freedompro.markdown [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+Check that the device has been correctly configured.  
+</div>
+
+## Preliminary Steps
+
+You will need to obtain an API key from [Freedompro Cloud](https://home.freedompro.eu/) to use this integration.
+
+1. Login to [Freedompro Cloud](https://home.freedompro.eu/).
+2. Click on your photo profile.
+3. Generate the API key,
```
Comment (frenck) [inline source/_integrations/freedompro.markdown]: `,`... ? Is there text missing?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/freedompro.markdown (hunk 2 above)
Comment (stefano055415) [reply source/_integrations/freedompro.markdown]: Our Products are bridges that control multiple type of accessories (light, switch, sensor,...). 
Should we remove all platforms except one for example light?

Comment (frenck) [reply source/_integrations/freedompro.markdown]: Yeah starting with just a light, switch or a sensor is fine 👍 
See also the comment on the main PR, which has a reference to our main documentation.

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_integrations/freedompro.markdown [hunk 5]
```diff
@@ -0,0 +1,33 @@
+---
+title: Freedompro
+description: Instructions for how to integrate Freedompro accessories within Home Assistant.
+ha_category:
+  - Light
+ha_release: 2021.3
```
Comment (frenck) [inline source/_integrations/freedompro.markdown]:
```suggestion
ha_release: 2021.7
```

File: source/_integrations/freedompro.markdown [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+ha_iot_class: Cloud Poll
+ha_config_flow: true
+ha_codeowners:
+  - '@stefano055415'
+ha_domain: freedompro
+---
+
+
```
Comment (frenck) [inline source/_integrations/freedompro.markdown]:
```suggestion

```

File: source/_integrations/freedompro.markdown [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+[Freedompro](https://freedompro.eu/), a company specialized in home automation, designs and manufactures products to make domotics affordable for everyone, installers and enthusiasts. [Freedompro Products](https://freedompro.eu/collections/easykon) are designed to be easy to use and practical and fast to install.
+
+This integration lets you control all [Freedompro](https://freedompro.eu/) accessories.
+
+
```
Comment (frenck) [inline source/_integrations/freedompro.markdown]:
```suggestion
```

File: source/_integrations/freedompro.markdown [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+1. Login to [Freedompro Cloud](https://home.freedompro.eu/).
+2. Make sure you have at least one accessory paired with the Freedompro Cloud before starting integration.
+3. Click on your photo profile.
+4. Generate the API key.
+
+
+## Configuration
+
+1. In the **Configuration** > **Integrations** menu, click **+** and then select "freedompro" from the pop-up menu.
+2. In the pop-up box, enter the API key you obtained from [Freedompro Cloud](https://home.freedompro.eu/).
```
Comment (frenck) [inline source/_integrations/freedompro.markdown]:
```suggestion
{% include integrations/config_flow.md %}
```

**Review by frenck — APPROVED** [review]
Thanks, @stefano055415 👍

**Pull request merged**